### DISCUSSION TITLE
Building ffmpeg-linux-x86_64 dependencies locally

### DISCUSSION THREAD

**Discussion by efifogel** [LizardByte/community#443] [Sunshine]
I'd like to build the ffmpeg-linux-x86_64 dependencies objects used by Sunshine locally. They, perhaps among other, comprise of ffmpeg libraries, and I'd like to apply some changes in their code.
I've cloned the build-deps repo.
I've checked out the master branch and initialized its submodules.
1. When running cmake I get errors from the commands that attempt to apply patches. I assume that this is not good; is it?
2. The cmake successfully ends (despite the errors above); then I make and obtain libcbs. Hoever, the directory `Sunshine_vdn/third-party/ffmpeg-linux-x86_64/lib` of my Sunshine clone after submodule initialization contains the libraries below. How do I build them?
libavcodec.a, libcbs.a, libSvtAv1Enc.a, libx264.a libavutil.a, libhdr10plus.a, libswscale.a, and libx265.a.

**Comment by ReenigneArcher** [comment]
> I've cloned the build-deps repo.

That's not enough. You need to build the changed code and push it to the `dist` branch. e.g. https://github.com/LizardByte/build-deps/tree/dist

This is all completely automated inside of GitHub, and probably not the easiest to handle in an environment other than that... depending on how well you understand GitHub workflows and such.

**Reply by ReenigneArcher** [reply]
The `build-deps` repo should probably be converted to a cmake project. If anyone wants to make an attempt at it, a PR would be welcome.

[a comment by an automated account is not shown]
